sources/packages/backend/libs/services/src/queue/dto/virus-scan.dto.ts

```diff
@@ -6,4 +6,10 @@ export interface VirusScanQueueInDTO {
 export interface VirusScanResult {
   fileProcessed: string;
   isInfected: boolean;
+  serverAvailability: string;
+}
+
+export interface VirusScanCode {
```

Comment:
As we discussed, unless there is a string reason we would not need this interface.